OTVision/pre_annotation.py

```diff
@@ -117,37 +117,18 @@ def _pre_annotate(
 
 
 def main(
-    file: Union[str, Path],
+    file: Path,
```

Comment:
Ups sorry, another thing: I think we should call it path instead of file. And then below we shoud check if path.is_file() or path.is_dir() and else raise Errors? Because otherwise main() would run through without errors, but uiser would wonder why nothing was pre annotated?

Reply:
Done!